packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -342,4 +408,48 @@ export default {
         display: block;
         font-weight: normal;
     }
+
+    .c-formField-icon {
+        svg {
+            position: absolute;
+            height: $form-input-iconSize;
+            width: $form-input-iconSize;
+
+            path {
+                fill: $color-content-subdued;
+            }
+        }
+    }
+
+    .c-formField-icon--disabled {
+        svg {
+            path {
+                fill: $color-content-disabled;
+            }
+        }
+    }
+
+    .c-formField-icon--small {
+        @include icon-position($form-input-iconPosition--small);
+    }
+
+    .c-formField-icon--medium {
+        @include icon-position($form-input-iconPosition--medium);
+    }
+
+    .c-formField-icon--large {
+        @include icon-position($form-input-iconPosition);
```

Comment:
I would be tempted to make the vertical and horizontal default position separate variables – as if one changes at some point in the future, we may want to change it independent of one another (so the left/right position may change, but the vertical large position may stay the same, or vice versa).

Something like `$form-input-icon-horizontalIndent` and `$form-input-icon-verticalIndent` would make sense I think?

Also, I'd make then setup the vertical variables so that the default is medium (as in the designs), so:

```scss
$form-input-icon-verticalIndent                  : 15px;
$form-input-icon-verticalIndent--small           : 11px;
$form-input-icon-verticalIndent--large           : 19px;

```